init.sh

```diff
@@ -80,6 +80,13 @@ cd srcomp-cli
 cd ..
 cd srcomp-screens
     bower install
+    python -c '
```

Comment:
Since this does rely on Python 3 style print call I'd be tempted to leave this as `python3`, certainly that should always work.